mate-panel/menu.c

```diff
@@ -562,11 +562,18 @@ show_item_menu (GtkWidget      *item,
 
 	gtk_menu_set_screen (GTK_MENU (menu),
 			     gtk_window_get_screen (GTK_WINDOW (panel_widget->toplevel)));
-
-	gtk_menu_popup (GTK_MENU (menu),
-			NULL, NULL, NULL, NULL,
-			bevent->button,
-			bevent->time);
+	/* Set up theme and transparency support */
+	GtkWidget *toplevel = gtk_widget_get_toplevel (menu);
+	/* Fix any failures of compiz/other wm's to communicate with gtk for transparency */
+	GdkScreen *screen = gtk_widget_get_screen(GTK_WIDGET(toplevel));
```

Comment:
Code styling. Can you please add spaces before brackets?
Some times you do that in this block and sometimes you don't :)